src/main/kotlin/no/nav/syfo/kafka/producers/brukernotifikasjoner/BrukernotifikasjonKafkaProducer.kt

```diff
@@ -60,84 +53,56 @@ class BrukernotifikasjonKafkaProducer(
         varselUrl: URL,
         smsContent: String?,
     ) {
-        val nokkelInput = buildNewNokkelInput(uuid, fnr)
-        val oppgave = buildNewOppgave(content, varselUrl, smsContent)
-
-        val record = ProducerRecord(
-            topicBrukernotifikasjonOppgave,
-            nokkelInput,
-            oppgave,
+        val varsel = createVarsel(
+            varseltype = Varseltype.Oppgave,
+            uuid = uuid,
+            fnr = fnr,
+            content = content,
+            varselUrl = varselUrl,
+            smsVarsling = true,
+            smsVarslingTekst = smsContent,
```

Comment:
Hvorfor sender vi med varslingstekst egentlig? Er det ikke en standard melding som sendes til brukeren på sms?

Reply:
I noen tilfeller overrider vi, dersom det er behov for en "viktig" tekst som ikke skal distribueres av dokdist. F.eks om du søker på BRUKERNOTIFIKASJON_AKTIVITETSKRAV_FORHANDSVARSEL_STANS_SMS_TEXT

Her er det behov for å ha noe mer "urgent" enn standard SMS